locales/en.json

```diff
@@ -1650,6 +1652,11 @@
         "title": "Minting NFT Collection",
         "submit": "Minting NFT Collection..."
       },
+      "drop": {
+        "confirm": "Teleport Required Amount",
+        "action": "Claim free",
```

Comment:
always free?

Reply:
aren't all drops that we are doing atm free?

maybe it should be generative_drop , what do you think ?

Reply:
not sure about that one, yes for now they are free

Reply:
checked current implementation and its not being checked

dropcard.vue

![CleanShot 2023-12-16 at 15 54 25@2x](https://github.com/kodadot/nft-gallery/assets/44554284/6218e810-dc8f-44ff-93df-724f17dbe0f2)